### PR TITLE
l2tp_client: allow to set MAX_BROKERS from outside

### PR DESCRIPTION
**Pull request by scherke**
For certain environments 10 brokers are not enought. Allow to
overwrite this value.
make CFLAGS="-D MAX_BROKERS=254"

### CODE REVIEW

**Comment by RalfJung** [comment]
Could you add this in some suitable place in the documentation?

File: docs/client.rst [hunk 1]
```diff
@@ -23,3 +23,15 @@ package. Source code for such OpenWrt package can be `found here`_.
...
 You can add the whole repository as an OpenWrt feed and add package to your firmware.
+
+Configuration
+-------------
+
+* **MAX_BROKERS** ``default: 10`` Maximum number of brokers that can be handled in a single process.
+
+    make CFLAGS="-D MAX_BROKERS=20"
+
+or you can edit the ``Makeflie`` and add
```
Comment (RalfJung) [inline docs/client.rst]: There's no `Makefile` (at least none that anyone should edit by hand), the client is using cmake.

Also, I am not sure what the right way is to format this with reStructuredText, but the "or you can edit" part is not appropriately indented like the part before (you can [see it rendered here](https://github.com/wlanslovenija/tunneldigger/blob/2ef04245c913a741826eea4e6715265f2131c83e/docs/client.rst)).

Comment (scherke) [reply docs/client.rst]: I mean the packages-opkg [``Makefile``](https://github.com/wlanslovenija/firmware-packages-opkg/tree/master/net/tunneldigger) 
I deleted the "or ..."

File: docs/client.rst [hunk 2]
```diff
@@ -23,3 +23,10 @@ package. Source code for such OpenWrt package can be `found here`_.
 .. _OpenWrt: https://openwrt.org/
 
 You can add the whole repository as an OpenWrt feed and add package to your firmware.
+
+Configuration
+-------------
+
+* **MAX_BROKERS** ``default: 10`` Maximum number of brokers that can be handled in a single process.
```
Comment (RalfJung) [inline docs/client.rst]:
```suggestion
* **MAX_BROKERS** (default: 10) Maximum number of brokers that can be handled in a single process.
```
I don't see a good reason to use code formatting here.

Comment (scherke) [reply docs/client.rst]: Okay, then that's how it's done. Do I have to change this in my pull request?

Comment (RalfJung) [reply docs/client.rst]: You should be able to press a button in GH's UI to accept my suggestion.

Comment (scherke) [reply docs/client.rst]: I don't have the button **Commit suggestion**. I will change my commit manually.

Comment (RalfJung) [reply docs/client.rst]: That's odd... it always worked for me. Thanks for doing it manually!

**Pull request merged**